### ISSUE TITLE
I want to start training again, may I ask to what is the convergence of loss reduction?

### ISSUE THREAD

**Issue by EdisonZhu33** [SWivid/F5-TTS#677]
### Checks

- [X] This template is only for question, not feature requests or bug reports.
- [X] I have thoroughly reviewed the project documentation and read the related paper(s).
- [X] I have searched for existing issues, including closed ones, no similar questions.
- [X] I confirm that I am using English to submit this report in order to facilitate communication.

### Question details

I want to start training again, may I ask to what is the convergence of loss reduction?

**Comment by SWivid** [comment]
Do you mean finetune or train from scratch with `start training again`?
- Could check for [training readme](https://github.com/SWivid/F5-TTS/blob/main/src/f5_tts/train/README.md)

And `the convergence of loss reduction` for the loss value reached with released pretrained model checkpoint?
- Could be found in [many existing issues](https://github.com/SWivid/F5-TTS/issues?page=2&q=is%3Aissue+loss) and also in discussion board

**Comment by sam4muzix** [comment]
> ### Checks
> 
> - [X] This template is only for question, not feature requests or bug reports.
> - [X] I have thoroughly reviewed the project documentation and read the related paper(s).
> - [X] I have searched for existing issues, including closed ones, no similar questions.
> - [X] I confirm that I am using English to submit this report in order to facilitate communication.
> 
> ### Question details
> 
> I want to start training again, may I ask to what is the convergence of loss reduction?

It depends upon number of samples , steps and then the total number of epochs can be resulting for exact convergence. 
What is your dataset size and how many samples u have ?

**Comment by ILG2021** [comment]
If the dataset is about 5000-6000 samples, how many steps should I train? The loss curse is very hard to make ensure when is convergence. Can we add a mel loss?

**Comment by ILG2021** [comment]
@SWivid
